.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,55 @@
+name: Publish Docker image
+
+on:  
+  push:
+    tags:
+      - '*'
```

Comment:
Basic trigger; if you push a tag it will build a docker image and tag it with the same github tag.